scripts/pytest/test_iam.py

```diff
@@ -0,0 +1,199 @@
+import time
+import pytest
+from arlas.cli.service import Service
+from conftest import USER_ADMIN
+from helper import create_collection, see_collection
+from variables import ANONYMOUS, COLLECTION1_ORG_1_PRIVATE, COLLECTION1_ORG_1_PUBLIC, INDEX_ORG1, INDEX_ORG2, ORG_1, ORPHAN, ORPHAN2, ORPHAN3, OWNER_ORG_1, OWNER_ORG_2, USER_ORG_1, USER_ORG_2
+
+
+# Test organisation creation
+def test_admin_create_org(fixture_init):
+    assert Service.create_organisation(USER_ADMIN, ORG_1).get("name") == ORG_1
+
+
+def test_orphan_not_create_org(fixture_init, fixture_orphans):
+    orphan, __, ___ = fixture_orphans
+    with pytest.raises(SystemExit):
+        assert Service.create_organisation(orphan, ORG_1)
+
+
+# Test add users
+def test_add_user_to_org_as_admin(fixture_init, fixture_org1_owner):
+    """as admin, I can add a user in org"""
+    oid, email, groups = fixture_org1_owner
+    assert OWNER_ORG_1 in list(map(lambda arr: arr[1], Service.list_organisation_users(USER_ADMIN, oid)))
+
+
+def test_add_user_to_org_as_owner(fixture_init, fixture_org1_owner):
+    """as an org owner, I can add a user in my org"""
+    oid, email, groups = fixture_org1_owner
+    Service.add_user_in_organisation(OWNER_ORG_1, oid, USER_ORG_1, [])
+    assert USER_ORG_1 in list(map(lambda arr: arr[1], Service.list_organisation_users(USER_ADMIN, oid)))
+
+
+@pytest.mark.parametrize("run_as",
+                         [OWNER_ORG_1,  # owner org 1 can not invite in org 2
+                          USER_ORG_1,   # user  org 1 can not invite in org 2
+                          USER_ORG_2,   # user  org 2 can not invite in org 2
+                          ORPHAN,       # orphan      can not invite in org 2
+                          ORPHAN2,      # orphan2     can not invite in org 2
+                          ORPHAN3,])    # orphan3     can not invite in org 2
+def test_not_add_user_to_org_as_not_owner(run_as, fixture_init, fixture_org1_owner_and_user, fixture_org2_owner, fixture_orphans):
+    """Test: as foreigner owner, foreigner user, user or orphan (with and without org-filter), I can not add a user in org"""
+    oid, __, __ = fixture_org2_owner
+    with pytest.raises(SystemExit):
```

Comment:
shouldn't be USER_ORG_2 ?

Reply:
In fact, doing for ORPHAN is even better, more combinaision tested. Thanks